### PR TITLE
Explain how to test pull requests and branches with our vagrant box

### PR DESCRIPTION
**Pull request by PierreRambaud**
Add a little block to explain why we use vagrant and add links to our [vagrant project](https://github.com/PrestaShop/vagrant).

### CODE REVIEW

File: src/content/1.7/contribute/contribute-by-testing-pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Contribute by testing Pull Requests and branches
+weight: 50
+menuTitle: Test Pull Requests and branches
+---
+
+# Why testing Pull Requests
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-by-testing-pull-requests.md:7]:
```suggestion
# Why test Pull Requests
```

File: src/content/1.7/contribute/contribute-by-testing-pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Tests determine the quality of a software. This process involves evaluating information that is related to a shop.
+
+To help on testing, we provide a [Vagrant](http://www.vagrantup.com/) configuration designed for testing branches and pull requests.
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-by-testing-pull-requests.md:11]:
```suggestion
To make testing easier, we provide a [Vagrant](http://www.vagrantup.com/) configuration designed for testing branches and pull requests.
```

Maybe explain what vagrant is and why you can use it.

File: src/content/1.7/contribute/contribute-by-testing-pull-requests.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+If you're using Windows, we recommend you to download [Git bash](https://git-scm.com/downloads) and execute all commands under this shell.
+
+## Why Vagrant
+
+We use Vagrant because of its flexibility and portability.
+With it, you can control every aspect of the machine, and help us by having a similar environment to reproduce issues.
+
+## Getting started
+
+Go to [https://github.com/PrestaShop/vagrant](https://github.com/PrestaShop/vagrant) and follow instructions.
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-by-testing-pull-requests.md:37]: This probably needs to be replaced with a step by step guide with screenshots.

File: src/content/1.7/contribute/contribute-by-testing-pull-requests.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+Tests determine the quality of a software. This process involves evaluating information that is related to a shop.
+
+To make testing easier, we provide a [Vagrant](http://www.vagrantup.com/) configuration designed for testing branches and pull requests.
+
+## What is installed
+
+- Debian 10 Buster
+- Git
+- Multiple PHP versions (7.1, 7.2, 7.3, 7.4)
```
Comment (Progi1984) [inline src/content/1.7/contribute/contribute-by-testing-pull-requests.md:17]:
```suggestion
- Apache
- Multiple PHP versions (7.1, 7.2, 7.3, 7.4)
```

File: src/content/1.7/contribute/contribute-by-testing-pull-requests.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+## Getting started
+
+The process is pretty simple.
+
+First, you must clone the repository [https://github.com/PrestaShop/vagrant](https://github.com/PrestaShop/vagrant).
+
+Second, execute the script `run.sh` (or `run.bat` if you're under Windows) and follow instructions.
+
+Most of the time, there is a prompt asking for a network interface, remember it's the communication between the virtual machine and our network card.
```
Comment (Progi1984) [inline src/content/1.7/contribute/contribute-by-testing-pull-requests.md:43]:
```suggestion
Most of the time, there is a prompt asking for a network interface, remember it's the communication between the virtual machine and the network card.
```

**Comment by Progi1984** [comment]
Thanks @PierreRambaud

**Pull request merged**